build/configs/esp32_DevKitC/scripts/target/esp_common.cfg

```diff
@@ -0,0 +1,116 @@
+# Common ESP chips definitions
+
+if { [info exists ESP_RTOS] } {
+	set _RTOS "$ESP_RTOS"
+} else {
+	set _RTOS "FreeRTOS"
```

Comment:
do we need to remove those RTOS config?

Reply:
I bring this file from openocd-esp32 without any change, the RTOS debugging supports FreeRTOS only, so I put `set ESP_RTOS none` in upper esp32.cfg file.